include/session/config/groups/members.hpp

```diff
@@ -230,11 +229,11 @@ struct member {
         // If the member is promoted then we assume they had accepted the invite and return the
         // relevant promoted status
```

Comment:
Just because we can now invite as promote
```suggestion
        // If the member is promoted then we assume we return the relevant promoted status
```